src/main/java/by/exadel/internship/controller/LocationController.java

```diff
@@ -0,0 +1,38 @@
+package by.exadel.internship.controller;
+
+import by.exadel.internship.dto.locationDTO.CityDTO;
+import by.exadel.internship.dto.locationDTO.CountryDTO;
+import by.exadel.internship.service.LocationService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/location")
+@Api(tags = "Endpoints for Location")
+public class LocationController {
+    private final LocationService locationService;
+
+    @GetMapping("/countries")
+    @ApiOperation("return list of countries")
+    public List<CountryDTO> getAllCountries() {
+        return locationService.getAllCountries();
+    }
+
+
+    @GetMapping("/countries/name")
+    @ApiOperation("return country by name")
+    public CountryDTO getCountryByName(@RequestParam(value = "name") String name){
+        return locationService.getCountryByName(name);
+    }
+    @GetMapping("/cities/name-country-city")
+    @ApiOperation("return city by name")
+    public List<CityDTO> getCityByName(@RequestParam(value = "countryName") String countryName,
```

Comment:
Why do we need this method, in fact we will only get the name of the city, which we sent

Reply:
It is redone